py-server/history_tutor/history_tutor.py

```diff
@@ -1,11 +1,58 @@
-def load_file(filename):
-    with open(filename, "r") as file:
-        return file.read()
-
-    history_tutor_context = prompts.CONTEXT_HISTORY_TUTOR
-    lister_and_carbolic_acid_context = load_file(
-        "./history_tutor/lister_and_carbolic_acid.md"
-    )
-    history_tutor_context = history_tutor_context.replace(
-        "%%CONTEXT%%", lister_and_carbolic_acid_context
-    )
+import time
+from langchain_google_vertexai import ChatVertexAI
+from langchain.prompts import PromptTemplate
+from langchain.chains import LLMChain
+from langchain_core.output_parsers import JsonOutputParser
+import json
+from langchain_core.messages import AIMessage, SystemMessage, HumanMessage
+
+PROMPT = """
```

Comment:
nit: remove extra newlines

Reply:
I removed one, I can't see from the snippet if I got the right one though.